src/idiom284ex2.fz

```diff
@@ -0,0 +1,23 @@
+ex284 is
+
+  n := 5
+
+  # This is another approach defining the list
+  a := (0 : nil).cycle.take n
+  say a
+
+  # This is another approach defining the list
```

Comment:
First define a list with one element and a generator function for successive elements. Then take n elements